SingularityUI/app/components/requestDetail/TaskHistoryTable.jsx

```diff
@@ -34,6 +34,18 @@ class TaskHistoryTable extends Component {
     };
   }
 
+  componentDidMount() {
+    const { requestId, initialPageNumber } = this.props;
+    if (initialPageNumber) {
```

Comment:
How is the data loaded if there is no initialPageNumber? The code for both cases should probably be in a single place.